Add Dutch translations store screenshot titles

keunes opened the pull request.

ByteHamster commented:
Thanks! Just generated the images (with the old screenshots), so you can see a preview. If you are happy with them, this can be merged.

<img width="200" src="https://user-images.githubusercontent.com/5811634/78010761-090be900-7343-11ea-9968-5a7f69fc9597.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/78010777-0c9f7000-7343-11ea-9458-70dbcdbbed2c.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/78010782-0dd09d00-7343-11ea-845a-cf787c94d181.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/78010788-0f9a6080-7343-11ea-926c-4ab11c600529.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/78010797-12955100-7343-11ea-9a77-0d6b49535458.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/78010802-14f7ab00-7343-11ea-87dd-f46a26f6c145.png" />

keunes commented:
Thanks for creating those. I spotted & corrected two mistakes, replaced 'thousands' by 'hundreds'  (the former sounds a bit dramatic in Dutch) and moved one word to the next line. Would you mind regenerating images 1 and 6, just to double-check they look ok?

Then, three comments beyond this specific PR:
- Image 2 seems to have a different background colour than the rest
- I like the texts, but it bugs me that only image 2 is not an action (do a/b/c) but descriptive (function x with y). Maybe `Powerful player\nwith sleep timer` could be prepended with 'Enjoy the'?
- I think it'd be great if these could be translated also in other languages, but I understand this requires stricter control. Would it be possible to create a new resource on Transifex that requires your or my approval?

ByteHamster commented:
> Image 2 seems to have a different background colour than the rest

On my screen, they look the same. All images are created from the same template (using a script), so this must be an optical effect.

> I like the texts, but it bugs me that only image 2 is not an action (do a/b/c) but descriptive (function x with y). Maybe Powerful player\nwith sleep timer could be prepended with 'Enjoy the'?

Good point. I want to fit the texts in two lines. Maybe something like "Sleep well \nwith the sleep timer"? In German, I translated them a bit differently. There, the automatic download string is so long that I removed the action. I just wrote "Automatisches Herunterladen" (automatic download).

> I think it'd be great if these could be translated also in other languages, but I understand this requires stricter control. Would it be possible to create a new resource on Transifex that requires your or my approval?

I think even with an approval, translators should see the screenshots before accepting the change. That is not easy to do on Transifex. It is probably easier to write a post on the Transifex team forum and do the translations "manually".

> Would you mind regenerating images 1 and 6, just to double-check they look ok?

<img width="250" src="https://user-images.githubusercontent.com/5811634/78770276-a17f1a80-798e-11ea-801e-709d35974ed1.png" /> <img width="250" src="https://user-images.githubusercontent.com/5811634/78770292-a643ce80-798e-11ea-94f7-3e81d0658f4d.png" />

---

I was thinking about switching screenshots 3 and 4 because Google Play only displays the first 3 screenshots (for others, you need to scroll). 4 has a more interesting layout and after doing my usability studies, I have the feeling that users are pretty obsessed with themes :D

keunes commented:
> On my screen, they look the same. All images are created from the same template (using a script), so this must be an optical effect.

Firefox's colour picker tells me they're different :) Would be odd if just that one is consistently (across sessions) rendered incorrectly. Maybe sth went odd when the script saved the image?

> I want to fit the texts in two lines. Maybe something like "Sleep well \nwith the sleep timer"?

It's a bit of a word play, but what about "Play smart\nwith a sleep timer". Or - I'm thinking the sleep timer may not be the most attractive - "Play smart with\nyour speed of choice"? (how I would translate those is another question). 'Sleep well with sleep timer' has some redundancy in it.

> I think even with an approval, translators should see the screenshots before accepting the change. That is not easy to do on Transifex. It is probably easier to write a post on the Transifex team forum and do the translations "manually".

Like any translations, it is a matter of incremental improvements (like we're doing now with Dutch). I think the main additional criteria for the screenshots are a) length and b) marketing feel, which both are relatively easy to check, even without having the images. Would it be possible to automatically push/pull strings from Transifex, let GitHub create the images, and store them in develop or sth for the translators to review, and that only then they're (manually) moved to master?

> I was thinking about switching screenshots 3 and 4
👍

ByteHamster commented:
> Firefox's colour picker tells me they're different :) Would be odd if just that one is consistently (across sessions) rendered incorrectly. Maybe sth went odd when the script saved the image?

You mean the blue background, right? On my PC, the color pickers of Chrome and Firefox both display the same blue background color for all images. Pretty strange :D

> 'Sleep well with sleep timer' has some redundancy in it.

Hmm that's right.

> I'm thinking the sleep timer may not be the most attractive - "Play smart with\nyour speed of choice"?

I have no idea what is more attractive... Never used another podcast app. Others probably also have a speed toggle but do they have a sleep timer?

> Would it be possible to automatically push/pull strings from Transifex, let GitHub create the images, and store them in develop or sth for the translators to review, and that only then they're (manually) moved to master?

Hmm I don't think it is easy to do. Additionally, I want to change the screenshots as infrequently as possible because they need a lot of space and therefore bloat the git history forever. I do not think that translators would actually look at the generated images, wherever they are generated. We probably need personal communication to really get them right.

I don't have a good idea on how to make the screenshot translations more automated :/

keunes commented:
By the way, two other screens I liked when checking competitors:
<img src="https://lh3.googleusercontent.com/bsbyJCzvD_3PIGkV-zfvO6FxV6N7fz8o07BqTCSNoL-ey17iQIMwk-fwIXFX2k2TvAim=w1296-h643">
(we should also mention Android Auto)

<img src="https://lh3.googleusercontent.com/4RBVYRyAj72T4scxdDJCKkItFzrh74Vo_pQFu0APr2HeN1wI-RB7TZoJ1EOeGagC3Ec=w250" width=250>
(we should note: free, no ads, open source)

keunes commented:
> You mean the blue background, right? On my PC, the color pickers of Chrome and Firefox both display the same blue background color for all images. Pretty strange :D

Yup, the blue background. And: wtf, in Chrome the colour picker identifies yet another colour, but at least it's the same for all images. Anyway, not really important.

> Others probably also have a speed toggle but do they have a sleep timer?

Well, of the 8 AP competitors I have in my bookmarks, all but one have an icon that indicates a sleep timer or explicitly mention it in their visuals. So it's definitively not unique. Personal I feel 'speed' is more relevant as it's a more basic, important feature for which there's no work-around, whereas the sleep timer feels more like a gimmick (I could also just press pause right before falling asleep). Hence my suggestion.

Dutch translations are ready now (unless EN player text is updated).

> We probably need personal communication to really get them right.

True. I thought this could be done by a language coordinator with you as maintainer, after images were created and checked. But I guess you're right in many cases there is no 'coordinator' who would check the images unless told so one on one.

ByteHamster commented:
> Personal I feel 'speed' is more relevant as it's a more basic, important feature for which there's no work-around

Feel free to change the text in the English (and Dutch) file :)

ByteHamster commented:
Maybe we should reduce the font size a bit (to improve the padding of `#4`) but `#2` is definitely too long ;)

<img width="200" src="https://user-images.githubusercontent.com/5811634/79053537-77538400-7c3e-11ea-99a1-c2a52cb3301e.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/79053540-7a4e7480-7c3e-11ea-8ef2-43978a0cc821.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/79053541-7b7fa180-7c3e-11ea-8529-49a71523f80b.png" />
<img width="200" src="https://user-images.githubusercontent.com/5811634/79053542-7e7a9200-7c3e-11ea-86e1-95bbe98684f9.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/79053543-7fabbf00-7c3e-11ea-8626-b88c532b8e56.png" /> <img width="200" src="https://user-images.githubusercontent.com/5811634/79053546-81758280-7c3e-11ea-8800-44d105782226.png" />

keunes commented:
@ByteHamster in #2 we can break one word up: 'Speel slim met afspeel-\nsnelheid naar keuze'

ByteHamster commented:
Thanks but I reduced the font size, so this is no longer needed :)
